blueocean-dashboard/src/main/less/forms/index.less

```diff
@@ -9,15 +9,17 @@
     justify-content: center;
 
     .Alert {
-        min-width: 400px;
```

Comment:
I made the experience that for xbrowser you should have both. IE being especially special.